### ISSUE TITLE
Error while trying to import library

### ISSUE THREAD

**Issue by weisheng-homage** [Vonage/vonage-node-sdk#855]
I'm seeing an error while trying to import the library, I'm on the most recent version

## Expected Behavior
Should not see an error while importing the library.

## Current Behavior
A bunch of error 
```
vonage_broke/.pnp.cjs:9418
      Error.captureStackTrace(firstError);
            ^

Error: Qualified path resolution failed: we looked for the following paths, but none could be accessed.

Source path: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/lib/user.js
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/lib/user.js.js
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/lib/user.js.json
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/lib/user.js.node
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/.js
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/.json
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/.node
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/index.js
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/index.json
Not found: vonage_broke/.yarn/cache/@vonage-users-npm-1.1.0-3915acba3b-2732d1f1a4.zip/node_modules/@vonage/users/index.node

Require stack:
- vonage_broke/.yarn/cache/@vonage-server-sdk-npm-3.8.0-1e78590b7f-2d36a79095.zip/node_modules/@vonage/server-sdk/dist/vonage.js
- vonage_broke/.yarn/cache/@vonage-server-sdk-npm-3.8.0-1e78590b7f-2d36a79095.zip/node_modules/@vonage/server-sdk/dist/index.js
- vonage_broke/run.js
    at Function.require$$0.Module._resolveFilename (vonage_broke/.pnp.cjs:9418:13)
    at Function.require$$0.Module._load (vonage_broke/.pnp.cjs:9268:42)
    at Module.require (node:internal/modules/cjs/loader:1067:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (vonage_broke/.yarn/cache/@vonage-server-sdk-npm-3.8.0-1e78590b7f-2d36a79095.zip/node_modules/@vonage/server-sdk/dist/vonage.js:13:17)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Object.require$$0.Module._extensions..js (vonage_broke/.pnp.cjs:9462:33)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.require$$0.Module._load (vonage_broke/.pnp.cjs:9299:14)
```

## Possible Solution
- it seems like the @vonage/users library was not compiled for the release hence the issue 
- also the recent release is overwriting the latest version, hence there is no option for us to rollback to an older working release.

## Steps to Reproduce (for bugs)
Sample minimal code to reproduce the issue here https://github.com/weisheng-homage/vonage_broke/tree/main

## Context
I won't be able to build anything at all without able to import this library

## Your Environment

* SDK Version: 3.8.0
* Node Version: (e.g. Node 10.18): 16.19.0
* Operating System and version: macos 13.0.1

**Comment by matheusisquierdo** [comment]
I had the same problem, for while I fixed the 3.7.2 version and solve de problem

**Comment by dragonmantank** [comment]
Could you try clearing your NPM or Yarn cache and try again?

**Comment by weisheng-homage** [comment]
thanks, looks like this is fixed in https://github.com/Vonage/vonage-node-sdk/pull/856
closing this

for anyone who is still facing this issue, you can fix your dependency for @vonage/users to 1.1.1 or @vonage/server-sdk to 3.8.1

edit: additional comments on how to get this fix on end user project
